lib/makeserver.js

```diff
@@ -37,8 +71,16 @@ module.exports = function(options) {
     app.use(compression());
     app.use(cors());
     app.disable('etag');
-    if (options.verbose) {
-        app.use(require('morgan')('dev'));
+    if (options.verbose || isMultiTenant) {
+      const morgan = require('morgan');
+      if (isMultiTenant) {
+        morgan.token('host', function(req, res) {
+          return req.hostname;
+        });
+        app.use(morgan(`:host :remote-addr - :remote-user [:date[clf]] ":method :url HTTP/:http-version" :status :res[content-length] ":referrer" ":user-agent"`));
```

Comment:
Should we add response time here also?

Reply:
https://github.com/TerriaJS/terriajs-server/pull/123/commits/c37a4fa037ad5f0a3057f100456a4145882a9f48#diff-a55b4af4fd0517d556660d3d45d52ca74973648f5424eae9df3ebb1a5dfe60ea

Reply:
Added!

Reply:
(from the oct 22 commit)